### ISSUE TITLE
Page-loader doc

### ISSUE THREAD

**Issue by evilz** [MangelMaxime/Fulma#54]
When navigating to Page loader extension, the loader is shown quickly.
need a display none or something like this ?

**Comment by MangelMaxime** [comment]
In fact, it's because the loader is being hidden :)
The loader have some CSS transition and that's the cause of this "visual bug".

I'm not sure how to fix it.

**Comment by MangelMaxime** [comment]
In fact, it's because the loader is being hidden :)
The loader have some CSS transition and that's the cause of this "visual bug".

I'm not sure how to fix it.

One solution could be to move the `if` condition to the loader level

**Comment by MangelMaxime** [comment]
One solution could be to move the `if` condition to the loader container level.

But it's feels wrong/odd to do that :)

https://github.com/MangelMaxime/Fulma/blob/master/docs/src/FulmaExtensions/PageLoader/View.fs#L18-L20

**Comment by alfonsogarciacaro** [comment]
You may be able to fix it by not displaying the element at all until it's initialized. For that you have to add a `IsInitialized` property or similar to the model (the code below means you have to use `yield` for other children as well, but you could do it differently):

```fsharp
if model.IsInitialized then
    yield PageLoader.pageLoader
        [ yield PageLoader.Types.Color model.Color
          if model.IsLoading
              yield PageLoader.isActive ]
```

**Comment by MangelMaxime** [comment]
@alfonsogarciacaro I was thinking like you, and I tried your solution but this doesn't work because, I never reset the models in the docs site. Their are initialized at the launch of the app and after, their are only updated. (I don't call `init` again).

I tried to set the condition on the container level but, this make the loader appear/disappear in one block instead of the animation. (when it go from top to bottom and vice versa).

Finally, if I force React to redraw the "part of DOM" with the page loader we don't have the visual bug. If we don't force react to redraw, then it only *patch* the DOM which trigger the animation.
